### ISSUE TITLE
Membership Coordinator can sort New Members list

### ISSUE THREAD

**Issue by judytuna** [doubleunion/arooo#292]
https://app.doubleunion.org/admin/new_members

As the membership coordinator, on the "new members" admin page, I'd like to be able to see new members who've set up their dues payments at the top, so I can add them to Slack and the Google group.

Consider: tracking for whether a new member has come to an orientation? (This may be redundant with becoming a keymember!)

**Comment by judytuna** [comment]
Also consider other New Members page improvements: 

- I'd like to be able to press a button for the case where a new member leaves the "google-services-friendly email address" email blank and I see that they're using a gmail address, so I'm assuming it's okay to add it to the mailing list. pressing this new button would copy the existing address into that field, so that i can press "Setup Finished" and move them out of the "New Member" page
- differentiate between "haven't set up dues yet" and "waiting on scholarship" so that if someone is waiting for scholarship setup, the webapp doesn't email them saying "hey please finish your setup!" (or we can add text to that reminder email saying "unless you're waiting for a scholarship, in which case you're all good and you'll hear from the board in the next two weeks) 
  - reminder emails: https://github.com/doubleunion/arooo/tree/master/app/views/new_members_mailer

**Comment by compwron** [comment]
This is great! We should definitely do these things.

**Comment by compwron** [comment]
I am going to split this into multiple smaller issues and close this one. 
https://github.com/doubleunion/arooo/issues/344
https://github.com/doubleunion/arooo/issues/345
https://github.com/doubleunion/arooo/issues/346
